site/include/form.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+
+$FormErrors = array();
+
+function TextBox($FieldName){
+	global $FormErrors;
+	echo"<input type='text' value='".@$_REQUEST[$FieldName]."'
+	name='$FieldName' />";
+	echo"$FormErrors[$FieldName]";
+}
+
+function EditTextBox($Value, $FieldName){
+	global $FormErrors;
+	echo"<input type='text' value='$Value' name='$FieldName' />";
+	echo"$FormErrors[$FieldName]";
+}
+
+function ReadOnlyTextBox($Value, $FieldName){
+	global $FormErrors;
+	echo"<input type='text' value='$Value' name='$FieldName' readonly />";
+	echo"$FormErrors[$FieldName]";
+}
+
+function DropDown($FieldName, $Option1, $Option2, $Option3, $Option4){
```

Comment:
As a rule of thumb when programming, there are only three number: 0, 1, and >1. If you ever find yourself handling more than one instance of something, you should almost always be able to make it handle an arbitrary number.

In this case, you could take an array of options instead of only allowing four options in a dropdown list.

Reply:
Along these lines, whatever format you want the options in might not be obvious, so this is a good time to leave a comment above the function that explains how the options should be formatted. Your future self will thank you.